plugins/@grouparoo/bigquery/src/lib/validateQuery.ts

```diff
@@ -0,0 +1,24 @@
+export function validateQuery(sql: string) {
+  const lowerCaseSQL = sql.toLowerCase();
+
+  if (lowerCaseSQL.indexOf(";") >= 0) {
+    throw new Error("only provide a single query");
+  }
+
+  if (lowerCaseSQL.indexOf("--") >= 0) {
```

Comment:
Might also want to check for "#" https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#comments